DRAFT_RELEASE_NOTES.md

```diff
@@ -26,6 +43,7 @@ To reduce burden of upgrading aiSSEMBLE, the Baton project is used to automate t
 | upgrade-tiltfile-aissemble-version-migration               | Updates the aiSSEMBLE version within your project's Tiltfile                                                                                                                                                                                           |
 | upgrade-v2-chart-files-aissemble-version-migration         | Updates the helm chart dependencies within your project's deployment resources (<YOUR_PROJECT>-deploy/src/main/resources/apps/) to use the latest version of the aiSSEMBLE                                                                             |
 | upgrade-v1-chart-files-aissemble-version-migration         | Updates the docker image tags within your project's deployment resources (<YOUR_PROJECT>-deploy/src/main/resources/apps/) to use the latest version of the aiSSEMBLE                                                                                   |
+| python-linting-migration                              | Updates the root `pom.xml` file to include Habushu in the plugin dependency and disables build error caused by linting. |
```

Comment:
A: Might make the description "Updates Habushu configuration to disable build failures when linting issues are detected."

Reply:
Updated with your suggestion, thanks Emily!